test/games/dread/exporter/test_dread_game_exporter.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+from unittest.mock import MagicMock, ANY
+
+import open_dread_rando.dread_patcher
+import pytest
+
+from randovania.games.dread.exporter.game_exporter import DreadGameExporter, DreadGameExportParams, DreadModPlatform
+from randovania.lib import json_lib
+
+
+@pytest.mark.parametrize("patch_data_name", ["starter_preset", "custom_start"])
+def test_export_game(test_files_dir, mocker, patch_data_name: str, tmp_path):
+    # Setup
+    def validate_schema(input_path: Path, output_path: Path, configuration: dict, status_update):
+        open_dread_rando.dread_patcher.validate(configuration)
+        status_update(1.0, "Finished")
+
+    mock_patch: MagicMock = mocker.patch("open_dread_rando.patch_with_status_update", side_effect=validate_schema)
+
+    patch_data = json_lib.read_path(test_files_dir.joinpath("patcher_data", "dread", f"{patch_data_name}.json"))
+
+    exporter = DreadGameExporter()
+    export_params = DreadGameExportParams(
+        spoiler_output=None,
+        input_path=tmp_path.joinpath("input_path"),
+        output_path=tmp_path.joinpath("output/path"),
```

Comment:
Is there a specific reason we use a `/` here? Could be `joinpath("output", "path")` in that case?

Reply:
Forgot and went with the easy way. Tbf for all our uses of paths, both separators work fine on Windows so it won't break anything.